firebase-app-distribution/src/main/java/com/google/firebase/app/distribution/FirebaseAppDistributionTesterApiClient.java

```diff
@@ -185,7 +185,7 @@ HttpsURLConnection openHttpsUrlConnection(
           ErrorMessages.NETWORK_ERROR, Status.NETWORK_FAILURE, e);
     }
     try {
-      httpsURLConnection.setRequestMethod(REQUEST_METHOD);
+      httpsURLConnection.setRequestMethod(REQUEST_METHOD_GET);
```

Comment:
Should this class also use the new HttpConnectionFactory?

Reply:
Ah, yeah probably. Maybe that's something we can add to the guiding principles, and update as we go? I think the principle would be: prefer injecting dependencies over mocking methods in the class-under-test.